plutus-core/plutus-core/src/PlutusCore/Parser/Builtin.hs

```diff
@@ -87,6 +89,25 @@ conData = do
         Right d  -> pure d
         Left err -> getSourcePos >>= customFailure . InvalidData (T.pack (show err))
 
+-- Serialised BLS12_381 elements are "0x" followed by a hex string of even
+-- length.  Maybe we should just use the usual bytestring syntax.
```

Comment:
I did it that way because you see hex strings represented like that quite a lot in the crypto world ([here](https://eips.ethereum.org/EIPS/eip-2537#specification) for example) and that made it easier to cut and paste (if you have to change the prefix it's easy to accidentally delete a digit without noticing).  It's not a big deal though.

Reply:
Strong +1 on using `0x` for BLS12_381 elements, no opinion on bytestrings.